src/App/MineAnsatte/SideBytter/SideBytter.tsx

```diff
@@ -31,32 +58,37 @@ const SideBytter = ({ className, antallSider, setParameterIUrl, plassering }: Pr
     }
 
     return (
-        <nav role="navigation" aria-label="Pagination Navigation" className={className}>
+        <nav role="navigation" aria-label={`Sidebytter, Nåværende side er ${nåVærendeSidetall}, bruk piltastene til å navigere`}
+             className={className}
+
+            >
             <div className="sidebytter">
                 {nåVærendeSidetall !==1 && <button
+                    onKeyDown={(e) => onSideendring(e.key)}
                     className="sidebytter__chevron"
                     id={'sidebytter-chevron-venstre-' + plassering}
                     onClick={() => {
                         setParameterIUrl('side',(nåVærendeSidetall - 1).toString());
                     }}
-                    aria-label={'Goto Page ' + (nåVærendeSidetall - 1).toString()}
+                    aria-label={'Gå til forrige side'}
                 >
                     <VenstreChevron type={'venstre'} />
                 </button>}
 
                 {(nåVærendeSidetall < 3 || antallSider < 4) && (
-                    <TreForste setParameterIUrl={setParameterIUrl} siderTilsammen={antallSider} nåVærendeSidetall={nåVærendeSidetall} />
+                    <TreForste onSideendring={onSideendring} elementIFokus = {elementIFokus} setParameterIUrl={setParameterIUrl} siderTilsammen={antallSider}  />
                 )}
                 {antallSider > 3 && nåVærendeSidetall > 2 && nåVærendeSidetall < antallSider - 1 && (
-                    <Midtdel nåVærendeSidetall={nåVærendeSidetall} setParameterIUrl={setParameterIUrl} siderTilsammen={antallSider} />
+                    <Midtdel onSideendring={onSideendring} elementIFokus = {elementIFokus} setParameterIUrl={setParameterIUrl} siderTilsammen={antallSider} />
                 )}
                 {antallSider > 3 && nåVærendeSidetall >= antallSider - 1 && (
-                    <TreSiste nåVærendeSidetall={nåVærendeSidetall} setParameterIUrl={setParameterIUrl} siderTilsammen={antallSider} />
+                    <TreSiste onSideendring={onSideendring}elementIFokus = {elementIFokus} setParameterIUrl={setParameterIUrl} siderTilsammen={antallSider} />
                 )}
                 <button
+                    onKeyDown={(e) => onSideendring(e.key)}
                     className={"sidebytter__chevron"}
-                    onClick={() => setParameterIUrl('side',(nåVærendeSidetall + 1).toString())}
-                    aria-label={'Goto Page ' + (nåVærendeSidetall - 1).toString()}
+                    onClick={() => setParameterIUrl('side',(nåVærendeSidetall+1).toString())}
+                    aria-label={'Gå til neste side'}
```

Comment:
Manger fokusmarkering på nesteknapp og forrigeknapp. Det leses opp i nvda at man kan bruke piltaster for å navigere på pagineringer, men det funker ikke. Funker det med pilnavigering i Voiceover?

Reply:
ja, det fungerer i voiceover :/ tror du vi er nødt å legge til role = tool bar igjen? ://